__mocks__/config-mock.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  app: {},
+  place_types: {},
+  place: {
+    place_detail: [
+      {
+        category: "someCategory",
+        fields: [
+          {
+            name: "test1",
+          },
+          {
+            name: "test2",
+          },
+        ],
+      },
+    ],
+  },
+  survey: {},
+  support: {},
+  pages: {},
+  story: {},
+  sidebar: {},
+  activity: {},
+  cluster: {},
+};
```

Comment:
I'm not sure if this is the best way to handle the config in testing. @modulitos -- I think you mentioned at one point we could use the `defaultflavor`'s config in testing, but at least initially I couldn't figure out how to make it work with the testing framework since it needs to pass through a series of webpack loaders before it's usable.

I need to research this a bit more, but in the meantime I put together this basic mock config.